pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue

```diff
@@ -15,6 +15,36 @@
         </table-row>
       </table>
     </UiCard>
+    <UiCard title="Ticket Inspection" class="mb-4">
+      <table>
+        <table-row>
+          <table-head-item>Ticket ID</table-head-item>
+          <table-row-item>Seat Group</table-row-item>
+          <table-row-item>Concession Type</table-row-item>
+          <table-row-item>Checked In</table-row-item>
+          <table-row-item v-if="anyTicketsChecked"
+            >Checked In By</table-row-item
+          >
+          <table-row-item v-if="anyTicketsChecked"
+            >Checked In At</table-row-item
+          >
+        </table-row>
+        <table-row v-for="ticket in booking.tickets" :key="ticket.id">
+          <table-head-item>{{ ticket.id }}</table-head-item>
```

Comment:
Also, just noticed from your screenshot, is there are reason why the ticket ID column and entries are bold?
I would have thought use `<table-head-item>` for each of the headers (Seat Group, Concession Type, etc.) and `<table-row-item>` for others?
```suggestion
          <table-head-item>Ticket ID</table-head-item>
          <table-head-item>Seat Group</table-row-item>
          <table-head-item>Concession Type</table-row-item>
          <table-head-item>Checked In</table-row-item>
          <table-head-item v-if="anyTicketsChecked"
            >Checked In By</table-row-item
          >
          <table-row-item v-if="anyTicketsChecked"
            >Checked In At</table-row-item
          >
        </table-row>
        <table-row v-for="ticket in booking.tickets" :key="ticket.id">
          <table-row-item>{{ ticket.id }}</table-row-item>
```

Reply:
This was just me thinking the first row had to be `<table-head-item>`, and subsequent ones `<table-row-item>`! Looks nicer changing it all over